assets/app.js

```diff
@@ -6,17 +6,21 @@
  */
 
 // any CSS you import will output into a single css file (app.css in this case)
-import "../assets/styles/components/global.css";
+import "../assets/styles/global.css";
+
 import "../assets/styles/components/banner.css";
 import "../assets/styles/components/navbar.css";
 import "../assets/styles/components/footer.css";
+import "../assets/styles/components/header.css";
+
 import "../assets/styles/others/faq.css";
```

Comment:
remove specific pages css files from app or it will unnecessarily load this css in every pages